src/components/playback/playersettingsmenu.js

```diff
@@ -238,6 +238,7 @@ function showWithUser(options, player, user) {
 
     return actionsheet.show({
         items: menuItems,
+        resolveOnClick: true,
         positionTo: options.positionTo
     }).then(function (id) {
```

Comment:
Should I also add a `catch` to handle the promise reject? I am not sure what more it will do than `console.error` the message. It's not much different than what it is doing now, but we will not have an unhandled promise showing up in the console like it currently does.

Reply:
I wouldn't worry about it. Currently none of our popups really handle the rejection when clicking outside. I imagine fixing them all would come as part of a larger refactor.